src/Driver/AbstractMySQLDriver.php

```diff
@@ -34,6 +35,10 @@ abstract class AbstractMySQLDriver implements VersionAwarePlatformDriver
     public function createDatabasePlatformForVersion($version)
     {
         $mariadb = stripos($version, 'mariadb') !== false;
+        if ($mariadb && version_compare($this->getMariaDbMysqlVersionNumber($version), '10.4.3', '>=')) {
+            return new MariaDb1043Platform();
+        }
```

Comment:
I've just merged #5946 which introduces a platform class for MariaDB 10.5.2. We need to adjust the inheritance chain accordingly.

Reply:
Updated. About to rebase and push.